swap/swap_test.go

```diff
@@ -112,13 +113,29 @@ func newTestBackend() *swapTestBackend {
 	// commit the initial "pre-mined" accounts (issuer and beneficiary addresses)
 	defaultBackend.Commit()
 
+	tokenAddress, _, token, err := contractFactory.DeployERC20Mintable(bind.NewKeyedTransactor(ownerKey), defaultBackend)
+	if err != nil {
+		log.Crit(err.Error())
+	}
+	defaultBackend.Commit()
+
+	_, err = token.Mint(bind.NewKeyedTransactor(ownerKey), ownerAddress, big.NewInt(1000000000000000000))
+	if err != nil {
+		log.Crit(err.Error())
+	}
+	defaultBackend.Commit()
+
 	// deploy a SimpleSwapFactoy
-	factoryAddress, _, _, _ := contractFactory.DeploySimpleSwapFactory(bind.NewKeyedTransactor(ownerKey), defaultBackend)
+	factoryAddress, _, _, err := contractFactory.DeploySimpleSwapFactory(bind.NewKeyedTransactor(ownerKey), defaultBackend, tokenAddress)
+	if err != nil {
+		log.Crit(err.Error())
```

Comment:
why just logging these errors and continue? Is the commit afterwards ok?

Reply:
See: https://github.com/ethersphere/swarm/pull/1964#discussion_r351346536